astro/src/content/docs/operate/roadmap/deprecation-policy.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Deprecation Policy
+description: Learn more about the FusionAuth deprecation policy.
+navcategory: admin
+section: operate
+subcategory: roadmap
+---
+
+## Overview
+
+FusionAuth is an architectural component integrated into your applications or systems. Such integration means that when changes to FusionAuth occur, it can negatively affect your application. As developers ourselves, we strive to maintain backwards compatibility. 
+
+However, there are times when features or functionality must be removed. FusionAuth has [extensive release notes](/docs/release-notes/), which are the source of truth for changes to the FusionAuth system, and any deprecated features will be listed there.
+
+Beyond adding them to the release notes, as a measure of our commitment to backwards compatibility, FusionAuth commits to certain steps and time before removing.
+
+### What Is A Feature
+
+Here are examples of features that will follow this policy.
+
+* A documented API endpoint or parameter
+* A documented way to authenticate against an API
+* A documented themeable page
+
+### What Is Not A Feature
+
+Anything not documented in the [FusionAuth technical documentation](/docs) is not considered a feature and may be removed without following this policy.
+
+## Deprecation Steps 
+
+Before removing a feature, the FusionAuth team takes the following steps:
+
+* The feature is marked as deprecated in the documentation.
+* In the first release where the behavior is deprecated, the deprecation is noted in the release notes.
+* The documentation deprecation note remains for three full releases.
+* In the fourth full release, the feature and corresponding documentation are removed.
+
+## Example Removal Timeline
+
+Suppose the current release is 1.52.0 and FusionAuth needed to remove feature ABC in the next release. Here is what that feature removal timeline would look like.
+
+| FusionAuth Version    | Feature ABC Changes | Release Notes Updates |
+| -------- | ------- | ------- |
+| 1.52.0  | No change | N/A |
+| 1.52.1  | No change | N/A |
+| 1.53.0  | Feature documentation marked as deprecated. | Deprecation announced in release notes. |
+| 1.53.1  | Feature documentation remains marked as deprecated. | N/A |
+| 1.53.2  | Feature documentation remains marked as deprecated. | N/A |
+| 1.54.0  | Feature documentation remains marked as deprecated. | N/A |
+| 1.55.0  | Feature documentation remains marked as deprecated. | N/A |
+| 1.56.0  | Feature and documentation removed. | Feature removal announced in release notes. |
+
+## Policy Exceptions
+
+If this is a change adding new functionality which requires changes to your integration, the FusionAuth team won't follow this policy. This only applies to deprecation and removal of functionality.
```

Comment:
Suggestion: (added may)

> If this is a change adding new functionality which may requires

Seems like if we are going to say this we should clarify. Which is to say, we will still do our best to document this in release notes to help the developer understand what needs to change, or if nothing needs to change based upon your implementation, but it falls outside of a true deprecation policy.